test/fixtures/index.js

```diff
@@ -121,6 +121,8 @@ module.exports = {
 
   inputV: getPath('vfile.v'),
 
+  inputJpgClahe: getPath('concert.jpg'),
```

Comment:
A reverse search for the test image suggests the source might be https://www.flickr.com/photos/mars_/14389236779/ - if so, please can you add a link to this in the test fixtures file so we know it's in the public domain.

Reply:
yes! that is where I found it and had the url copied to my clipboard to do so. Nice catch.